db/migrate/20210331000526_add_id_to_spartan_session_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddIdToSpartanSessionUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spartan_session_users, :id, :primary_key
```

Comment:
Join tables should not have ids

Reply:
It was the only way to please rubocop

Reply:
Why does Rubocop care? It's never flagged an issue with this table before

Reply:
The problem is that I need to access the first_checkin and second_checkin attributes and be able to update them when a user checks in. At least the second_checkin attribute.